packages/block-editor/src/components/block-list/block.js

```diff
@@ -261,7 +261,11 @@ function BlockListBlock( {
 	}, [ isFirstMultiSelected ] );
 
 	// Block Reordering animation
-	const animationStyle = useMovingAnimation( wrapper, isSelected || isPartOfMultiSelection, enableAnimation, animateOnChange );
+	// useMovingAnimation uses hooks and requires being called even with disabled animations.
+	let animationStyle = useMovingAnimation( wrapper, isSelected || isPartOfMultiSelection, enableAnimation, animateOnChange );
+
+	// Dismiss animation style if animations are disabled.
+	animationStyle = enableAnimation ? animationStyle : {};
```

Comment:
Why don't we do that in the hook itself?

Reply:
Moved within 029c0a8.